src/devices/Mcp3xxx/README.md

```diff
@@ -1,14 +1,128 @@
 # MCP3xxx family of Analog to Digital Converters
 
-Note: Currently untested on the Mcp33xx family.
-
 Some devices like the Raspberry Pi cannot read analog values directly so rely on  [analog to digital converters](https://en.wikipedia.org/wiki/Analog-to-digital_converter), like the ones available from Microchip in the Mcp3000, Mcp3200 and Mcp3300 ranges. These chips can be accessed as an [SPI device](https://en.wikipedia.org/wiki/Serial_Peripheral_Interface) or manually via raw GPIO pins.
 
- You can use these converters in your project to access analog devices. The sample [Reading Analog Input from a Potentiometer](samples/README.md) demonstrates a concrete example using the Mcp3008 class.
+You can use these converters in your project to access analog devices.
 
 The following fritzing diagram illustrates one way to wire up the Mcp3008, with a Raspberry Pi and a potentiometer.
 
-![Raspberry Pi Breadboard diagram](samples/rpi-trimpot_spi.png)
+![Raspberry Pi Breadboard diagram](rpi-trimpot_spi.png)
+
+## Documentation
+
+The sample is based on following resources:
+
+- [Analog Inputs for Raspberry Pi Using the MCP3008](https://learn.adafruit.com/reading-a-analog-in-and-controlling-audio-volume-with-the-raspberry-pi) 
```

Comment:
Note: this is a great way to have the enumeration and the links on documentations.